src/libraries/Directory.Build.targets

```diff
@@ -206,4 +206,9 @@
     <ExcludeFromPackage Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)' and '$(ExcludeCurrentNetCoreAppFromPackage)' == 'true'">true</ExcludeFromPackage>
     <ExcludeFromPackage Condition="'$(TargetFramework)' == '$(NetFrameworkCurrent)' and '$(ExcludeCurrentFullFrameworkFromPackage)' == 'true'">true</ExcludeFromPackage>
   </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(MSBuildProjectDirectory)\**\*.cs"
```

Comment:
```suggestion
    <None Include="$(MSBuildProjectDirectory)**\*.cs"
```

Reply:
The slash is required $(MSBuildProjectDirectory) doesnot have a trailing slash, and vs throws an error while trying to load this regex

Reply:
> Do not include the final backslash on this property.

Oh right :(

Reply:
I guess this won't cause trouble by including duplicates with the ones included in the Compile item?

Reply:
Those are excluded in the Exclude filter.

Reply:
Ah, I didn't see them as I just replied the question here and it didn't show all the lines 😄